roles/strimzi_kafka/tasks/install.yaml

```diff
@@ -0,0 +1,43 @@
+---
+- name: Install Strimzi Operator
+  kubernetes.core.k8s:
+    state: present
+    definition: "{{ lookup('url', item, split_lines=False) }}"
```

Comment:
for single item we could just have
```
    definition: "{{ lookup('url', kafka_install_url, split_lines=False) }}"
```
but as I see now we are not consistent in other places so can be refactored later